eahub/profiles/migrations/0004_auto_20190302_1907.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.7 on 2019-03-02 19:07
```

Comment:
Only one migration per PR, please, unless there's a specific reason it needs to be broken up.

Also, I think we should specify a [through model](https://docs.djangoproject.com/en/2.1/topics/db/models/#intermediary-manytomany) here. We don't need any extra fields now, but we're likely to later and it's hard to change.